src/bundle/Resources/public/js/scripts/admin.location.adaptive.tabs.js

```diff
@@ -1,147 +1,184 @@
-                const maxTotalWidth = primaryTabsList.offsetWidth - OFFSET_ROUNDING_COMPENSATOR;
-                const hiddenPrimaryTabs = [];
-                let currentWidth = moreTabWidth + activePrimaryTabWidth;
-
-                primaryTabs.forEach((tab) => tab.classList.remove('ez-tabs__tab--hidden'));
-                moreTab.classList.remove('ez-tabs__tab--hidden');
-
-                for (let i = 0; i < primaryTabs.length; i++) {
-                    const tab = primaryTabs[i];
-                    const tabLink = tab.querySelector('.nav-link');
-
-                    if (tabLink === activePrimaryTabLink) {
-                        continue;
-                    }
-
-                    const isLastTab = i === primaryTabs.length - 1;
-                    const allPreviousTabsVisible = hiddenPrimaryTabs.length === 0;
-                    const isTabNarrowerThanMoreTab = tab.offsetWidth < moreTab.offsetWidth + OFFSET_ROUNDING_COMPENSATOR;
-
-                    if (isLastTab && allPreviousTabsVisible && isTabNarrowerThanMoreTab) {
-                        break;
-                    }
-                    if (currentWidth + tab.offsetWidth + OFFSET_ROUNDING_COMPENSATOR > maxTotalWidth) {
-                        hiddenPrimaryTabs.push(i);
-                    }
-
-                    currentWidth += tab.offsetWidth + OFFSET_ROUNDING_COMPENSATOR;
-                }
-
-                primaryTabs.forEach((tab, index) => {
-                    tab.classList.toggle('ez-tabs__tab--hidden', hiddenPrimaryTabs.includes(index));
-                });
-                secondaryTabs.forEach((tab, index) => {
-                    tab.classList.toggle('ez-tabs__tab--hidden', !hiddenPrimaryTabs.includes(index));
-                });
-
-                moreTab.classList.toggle('ez-tabs__tab--hidden', !hiddenPrimaryTabs.length);
-            }
+        allAdaptiveItems.forEach((adaptiveItems) => {
+            adaptiveItems.adapt();
+        });
+    };
+    const positionPopupMenu = () => {
+        doc.querySelectorAll('.ibexa-tabs__popup-menu:not(.ibexa-tabs__popup-menu--hidden)').forEach((popupMenu) => {
+            const tabsContainer = popupMenu.parentElement;
+            const tabsList = tabsContainer.querySelector(SELECTOR_TABS_LIST);
+            const tabMore = tabsList.querySelector(':scope > .ibexa-tabs__tab--more');
+            const popupMenuAbsoluteLeft = tabMore.offsetLeft + tabsList.offsetLeft - popupMenu.offsetWidth + tabMore.offsetWidth + 20;
+
+            popupMenu.style.left = `${popupMenuAbsoluteLeft}px`;
         });
     };
-    const handleClickOutsideSecondaryMenu = (event) => {
-        doc.querySelectorAll(TABS_SELECTOR).forEach((primaryTabsList) => {
-            const moreTab = primaryTabsList.querySelector('.ez-tabs__tab--more');
+    const handleTabsConainterChange = () => {
+        if (frame) {
+            cancelAnimationFrame(frame);
+        }
+
+        frame = requestAnimationFrame(adaptTabsAndPopupMenu);
```

Comment:
I wonder if frame is correct name, maybe sth like animationFrame? As frame is quite vast term